src/Illuminate/Cache/DatabaseStore.php

```diff
@@ -98,29 +99,53 @@ public function __construct(ConnectionInterface $connection,
      */
     public function get($key)
     {
-        $prefixed = $this->prefix.$key;
-
-        $cache = $this->table()->where('key', '=', $prefixed)->first();
+        return $this->many([$key])[$key];
+    }
 
-        // If we have a cache record we will check the expiration time against current
-        // time on the system and see if the record has expired. If it has, we will
-        // remove the records from the database table so it isn't returned again.
-        if (is_null($cache)) {
-            return;
+    /**
+     * Retrieve multiple items from the cache by key.
+     *
+     * Items not found in the cache will have a null value.
+     *
+     * @return array
+     */
+    public function many(array $keys)
```

Comment:
https://redberry.international/laravel-10-features-and-updates/#mcetoc_1hfrpmpr11v
please put **: array** as return type

Reply:
and remove the dockblock @return pls

Reply:
I'd rather keep the same method signature as the interface we're implementing.

Reply:
If you are implementing an interface I agree, but I see this as a new function that is not in the interface.

Reply:
@tonysm  I apologize. I saw this commit https://github.com/laravel/framework/commit/00dfdf48a12a79408d2646ba9540e8636e270e51 and I will never ever open this subjet in the laravel community again.